helpers.lua

```diff
@@ -713,8 +713,7 @@ end
 -- Read result of a command after given it securely the password
 function secure_cmd_password(cmd, password, start)
     -- Check password validity
-    math.randomseed( os.time() )
-    local tmp_file = "/tmp/ssowat_"..math.random()
+    local tmp_file = os.tmpname()
```

Comment:
For anyone wondering, the permissions for the file created appears to be 600, which is safe :+1: